kedro-datasets/kedro_datasets/databricks/managed_table_dataset.py

```diff
@@ -7,16 +7,13 @@
 from typing import Any, Dict, List, Optional, Union
 
 import pandas as pd
-from kedro.io.core import (
-    AbstractVersionedDataSet,
```

Comment:
If https://github.com/kedro-org/kedro/pull/2746 goes through, this will also need to be handled.

Reply:
I'll do it preemptively